packages/components/src/draggable/README.md

```diff
@@ -8,9 +8,16 @@ Note that the drag handle needs to declare the `draggable="true"` property and b
 
 The component accepts the following props:
 
+### `attachToElementWrapper`: `boolean`
+
+Whether to attach the cloned element to the element's wrapper when using `elementId`. Setting to false will attach the cloned element to the `ownerDocument` body.
```

Comment:
I've reworded it, but retained reference to the id, as otherwise it could be confusing that `appendToOwnerDocument` defaults to `false` but is still in use when someone's using the experimental drag component. Happy to keep tweaking / feel free to go with different wording if it still feels off!